lib/delayed/backend/shared_spec.rb

```diff
@@ -574,7 +574,6 @@ def create_job(opts = {})
       it 're-schedules jobs after failing' do
         worker.work_off
         @job.reload
-        expect(@job.error).to_not be_nil
```

Comment:
Does it make sense to change this line `expect(@job.last_error).to_not be_nil` or do you think that’s sufficiently covered by the next two lines of the spec?